### PR TITLE
Add list example in the proper place

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #729
[Problem] List example wasn't in proper place according to documentation
[Solution]

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/presentation-views/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+    <title>
+        Wearable UI
+    </title>
+    <link href="../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+    <link href="../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
+    <link href="../../css/style.css" rel="stylesheet" />
+    <link href="../../css/style.circle.css" rel="stylesheet" />
+    <script src="../../js/circle-helper.js">
+    </script>
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/index.html]: Can we have those scripts tags in one line?
`<script src="../../js/circle-helper.js"></script>`

Having them in multiline may suggest that there is something inside but here it's empty.

File: examples/wearable/UIComponents/contents/presentation-views/index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <script src="../../lib/tau/wearable/js/tau.js">
+    </script>
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/index.html]: Ditto.

File: examples/wearable/UIComponents/contents/presentation-views/index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <header class="ui-header">
+            <h2 class="ui-title">Presentation views</h2>
+        </header>
+        <div class="ui-content">
+            <ul class="ui-list">
+                <li><a href="list/index.html">List</a></li>
+            </ul>
+        </div>
+    </div>
+
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/index.html]: Unnecessary line.

File: examples/wearable/UIComponents/contents/presentation-views/index.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+</head>
+
+<body>
+    <div class="ui-page" data-go-to-the-top-button="true">
+        <header class="ui-header">
+            <h2 class="ui-title">Presentation views</h2>
+        </header>
+        <div class="ui-content">
+            <ul class="ui-list">
+                <li><a href="list/index.html">List</a></li>
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/index.html]: I think others examples use multi line for this:
```
 <li>
    <a href="list/index.html">
        List
    </a>
</li>
```

File: examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html [hunk 5]
```diff
@@ -6,21 +6,21 @@
 	<title>
 		Wearable UI
 	</title>
-	<link href="../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
-	<link href="../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
-	<link href="../../css/style.css" rel="stylesheet" />
-	<link href="../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html:12]: Don't we need media queries?

Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html:12]: This file is removed.

File: examples/wearable/UIComponents/contents/presentation-views/list/index.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<link href="../../../css/style.circle.css" rel="stylesheet" />
+	<script src="../../../js/circle-helper.js"></script>
+	<script src="../../../lib/tau/wearable/js/tau.js"></script>
+</head>
+
+<body>
+    <div class="ui-page" data-go-to-top-button="true">
+		<header>
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/list/index.html:19]: Please check indents.

File: examples/wearable/UIComponents/contents/presentation-views/list/index.html [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+				List
+			</h2>
+        </header>
+        <div class="ui-content">
+			<ul class="ui-listview">
+				<li>
+					<a href="basic-list.html">
+						Basic List
+					</a>
+                </li>
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/presentation-views/list/index.html]: Ditto.

File: examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html (hunk 5 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html:12]: Looks like rename, do we need this file?

Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html:12]: Yeah technically this is rename, but I removed these media queries in the new version. And yes, we need this file.

File: examples/wearable/UIComponents/index.html [hunk 8]
```diff
@@ -30,6 +30,16 @@ <h2 class="ui-title">
 						Navigation elements
 					</a>
 				</li>
+				<li>
+					<a href="contents/content/index.html">
```
Comment (singa2000) [inline examples/wearable/UIComponents/index.html]: Now contents/content folder is removed.
Do you have a plan to add the folder?

File: examples/wearable/UIComponents/contents/presentation-views/list/index.html (hunk 6 above)
Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/presentation-views/list/index.html:19]: Done

File: examples/wearable/UIComponents/index.html (hunk 8 above)
Comment (korneliakobiela) [reply examples/wearable/UIComponents/index.html]: No, it was added by accident. Thank you for your comment

**Review by grzegorz-czajkowski — APPROVED** [review]

**Pull request merged**